docs/Design/SoftDetailedDes/MIS.tex

```diff
@@ -306,6 +306,77 @@ \subsubsection{Local Functions}
 
 \newpage
 
+\section{MIS of Authentication Module} \label{mAuthentication} 
+The Authentication module acts to "log-in" the user by providing the client with a JWS token.
+
+\subsection{Module}
+
+authentication
+
+\subsection{Uses}
+
+ Database Driver Module (Section \ref{mDBDriver}) 
+
+\subsection{Syntax}
+
+\subsubsection{Exported Constants}
+None
+
+\subsubsection{Exported Access Programs}
+
+\begin{center}
+\begin{tabular}{p{5cm} p{3cm} p{3cm} p{5cm}}
+\hline
+\textbf{Name} & \textbf{In} & \textbf{Out} & \textbf{Exceptions} \\
+\hline
+getToken & String, String & String & invalidCredentialsException \\
+verifyToken & String & String & invalidTokenException \\
+\hline
+\end{tabular}
+\end{center}
+
+\subsection{Semantics}
+
+\subsubsection{State Variables}
+None
+
+\subsubsection{Environment Variables}
+PrivateKey (for JWT)
```

Comment:
Think you need a `\\` if you want these two Environment Variables to appear on two different lines unless it's rendering correctly on your end.